Signal/Images.xcassets/file-icon-large.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "file-icon-large@1x.png",
```

Comment:
For large "hero" image assets that we want to resize based on device size, I like to use a single oversized asset and scale using UIImageView.